Funky-Chickens/bangazonAPI
# Cm faker data

I made the following changes:


1. fixed the syntax for all SQL statements to match AirBNB uniformity

2. added faker data for computers and created a place for the employees/computer join table data to live. 

3. fixed the dirname issue in build db


**I fixed the syntax of all the existing methods so that the code looks like one person wrote it, and made it easier to read and follow. I added more fake data so that we can start writing the app with all the necessary components.**

Steps to test:

check the syntax for all the methods in build-db.js. if all of the formats match (only contain values that are being pushed), then run node build-db.js in git bash and open sqlite to see if all columns in orders and training tables were populated, including "Computers"

feature ticket: null - refactoring
